### PR TITLE
feat: add accordeon component, modules to import images

### PR DESCRIPTION
**Pull request by NSVoronova**
1. Created accordeon component with animations and styles (in the future it will be necessary to adjust the styles to the style of the parent block)
2. Added modules to import images in project

### CODE REVIEW

File: src/components/Accordeon/Accordeon.tsx
Comment (KaterinaPogorelova) [inline src/components/Accordeon/Accordeon.tsx:1]: Animation looks cool, moves are smooth, nice! But when I created the task, I meant the entire frame layout, not just the accordion logic) But accordeon logic looks nice!

Comment (NSVoronova) [reply src/components/Accordeon/Accordeon.tsx:1]: Okay, I'll update my request later)

File: src/styledConstants.ts [hunk 1]
```diff
@@ -2,5 +2,58 @@ import styled from "styled-components";
 
 export const colors = {
   violet: "#8657DD",
```
Comment (KaterinaPogorelova) [inline src/styledConstants.ts:4]: Designers asked to do more colors categories like in their library (you can see more in my commit)

File: src/styledConstants.ts [hunk 2]
```diff
@@ -2,5 +2,58 @@ import styled from "styled-components";
...
+  violet5: "rgba(128,71,235,0.05)",
+  champagne: "#FAF9F2",
+  white: "#FFFFFF",
+  violet: "#8657DD",
+}
+
+export const StyledFlex = styled.div` // это можно дополнять условиями
+  display: flex;
+`
+export const StyledContent = styled.div` // Обертка для контента на странице
```
Comment (KaterinaPogorelova) [inline src/styledConstants.ts:25]: what about safe paddings?

File: src/styledConstants.ts [hunk 3]
```diff
@@ -2,5 +2,58 @@ import styled from "styled-components";
...
+  black: "#111111",
+  darkGray: "#999999",
+  green: "#65DC6A",
+  white: "#FFFFFF",
+  gradientG3: "linear-gradient(90deg, #65DC6A 0%, #7293A9 52.09%, #8047EB 106.81%)"
+};
+
+export const backgroundColors = { // девочки-дизайнеры просили разделить цвета текста и фона
+  black: "#111111",
+  violet20: "rgba(134, 87, 221, 0.20)",
```
Comment (KaterinaPogorelova) [inline src/styledConstants.ts:15]: You need to do colors in one style, preffered HEX

**Review by KaterinaPogorelova — APPROVED** [review]
okkk!

**Pull request merged**